packages/forms/src/elements/common/Fieldset.tsx

```diff
@@ -43,7 +43,7 @@ export const Fieldset = forwardRef<HTMLFieldSetElement, IFieldsetProps>((props,
 
   return (
     <FieldsetContext.Provider value={fieldsetContext}>
-      <StyledFieldset {...props} ref={ref} />
+      <StyledFieldset {...props} ref={ref} isCompact={props.isCompact} />
```

Comment:
Should isCompact be destructured out of `{ isCompact, ...props }` in the function signature so that it isn't applied to the `StyledFieldset` twice? Or is the `{...props}` sufficient on this line without `isCompact`?

Reply:
Oops, yeah this is not needed since props are already passed to `StyledFieldset`.